### PR TITLE
pythonPackages.afdko: 3.5.0 → 3.5.1

### PR DESCRIPTION
**Pull request by sternenseemann**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sternenseemann** [comment]
cc @roconnor for testing

**Comment by sternenseemann** [comment]
@GrahamcOfBorg build python3Packages.nototools

File: pkgs/development/python-modules/afdko/default.nix [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
-    sha256 = "0wid4l70bxm297xgayyrgw5glhp6n92gh4sz1nd4rncgf1ziz8ck";
+    sha256 = "1qg7dgl81yq0sp50pkhgvmf8az1svx20zmpkfa68ka9d0ssh1wjw";
   };
 
+  # Skip date-dependent test. See
+  # https://github.com/adobe-type-tools/afdko/pull/1232
+  # https://github.com/NixOS/nixpkgs/pull/98158#issuecomment-704321117
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/adobe-type-tools/afdko/pull/1232.patch";
```
Comment (marsam) [inline pkgs/development/python-modules/afdko/default.nix:25]: please use the commit patch
```suggestion
      url = "https://github.com/adobe-type-tools/afdko/commit/2c36ad10f9d964759f643e8ed7b0972a27aa26bd.patch";
```

Comment (sternenseemann) [reply pkgs/development/python-modules/afdko/default.nix]: done

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 100575 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>6 packages built:</summary>  <ul>    <li>noto-fonts-emoji</li>    <li>python37Packages.afdko</li>    <li>python37Packages.nototools</li>    <li>python38Packages.afdko</li>    <li>python38Packages.nototools</li>    <li>twitter-color-emoji</li>  </ul></details>

**Pull request merged**